detect_secrets/plugins/core/initialize.py

```diff
@@ -10,6 +10,7 @@
 from ..high_entropy_strings import HexHighEntropyString     # noqa: F401
```

Comment:
Below in this file is

https://github.com/Yelp/detect-secrets/blob/c0bea0b9b0dad74edf5d197eada36dfa64dd3e5a/detect_secrets/plugins/core/initialize.py#L92

you'll probably want to do this with `BasePlugin` as well.

cc @domanchi for confirmation

Reply:
`RegexBasedDetector` isn't imported here, so it won't be included anyway

Reply:
> You'll probably want to do this with `BasePlugin` as well.

Accurate, but for `RegexBasedDetector`.

```
if issubclass(value, BasePlugin) and value not in [BasePlugin, RegexBasedDetector]:
```